qiskit/transpiler/passes/layout/noise_adaptive_layout.py

```diff
@@ -71,6 +72,21 @@ def __init__(self, backend_prop):
         else:
             self.target = None
             self.backend_prop = backend_prop
+
+            if coupling_map:
+                # A backend might have more properties than qubits/gates in the configuration. This is a
+                # problem that the Target path should handle differently (by solving that possible
+                # inconsistency internally). For the non-target path, this is a possible solution.
+                # See https://github.com/Qiskit/qiskit/issues/7677
+                self.backend_prop.gates = filter(
+                    lambda ginfo: ginfo.gate == "cx"
```

Comment:
What about backends that use different entangling gates besides cx?

Reply:
i think the pass is already cx-specific https://github.com/Qiskit/qiskit/blob/fc74ab96b1732bfc4b37d32b830c11b14d0960f8/qiskit/transpiler/passes/layout/noise_adaptive_layout.py#L92 . But it is easy to fix by just removing that condition, in 6d9c214afc9340b5124be6fa861521651a4b8cb6